data/external/scrapers/public_transport.py

```diff
@@ -4,16 +4,11 @@
 from external.scraping_utils import _download_file, CACHE_PATH, cached_json
 
 MVV_GTFS_URL = "https://www.mvv-muenchen.de/fileadmin/mediapool/02-Fahrplanauskunft/03-Downloads/openData/mvv_gtfs.zip"
+MVV_HST_REPORT_URL = "https://www.mvv-muenchen.de/fileadmin/mediapool/02-Fahrplanauskunft/03-Downloads/openData/MVV_HSTReport2212.csv"  # train/tram stations + some bus stations
```

Comment:
This is named `MVV_HSTReport2212.csv` => Will we need to update this in the future?

Reply:
probably, yes